docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,371 @@
+    :alt: Aiven Console, active MirrorMaker 2 integration
+
+
+Create a data replication using MirrorMaker 2
+---------------------------------------------
+
+The next step in the migration journey is to create a data replication from the source cluster to the Aiven for Apache Kafka service named ``demo-kafka``. To create a replication you need to:
+
+* Create an alias for the source Apache Kafka cluster (the target alias ``kafka-target`` was defined during the creation of the MirrorMaker 2 service).
+* Define the replication follow.
+
+Create an alias for the source Apache Kafka cluster
+'''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create a MirrorMaker 2 replication flow, first create an alias to point to the source Kafka cluster. You defined the target alias ``kafka-target`` during the creation of the MirrorMaker 2 service, so use that.
+
+To create the alias with the `Aiven Console <https://console.aiven.io/>`_ you can follow the steps below:
+
+1. Navigate to the MirrorMaker 2 ``demo-mm2`` service page.
+2. Click on the **Integration** tab.
+3. Scroll until you reach the **External integrations** section.
+4. Select **Cluster for replication** within the available external integrations.
+
+    
+5. Select the endpoint name you defined in a previous step (``mySourceKafkaCluster``) and click **Continue**.
+
+   .. image:: /images/tutorials/kafka-migration/external-endpoint-integration.png
+    :alt: Aiven Console, MirrorMaker 2 alias definition for the external Apache Kafka endpoint integration
+
+6. Give the source cluster an alias name, for example ``kafka-source``, and click **Enable**.
+
+Once the steps are done, you should be able to see the two aliases, ``kafka-source`` and ``kafka-target`` defined as integrations in the ``demo-mm2`` service:
+
+.. image:: /images/tutorials/kafka-migration/source-target-integration-enabled.png
+    :alt: Aiven Console, MirrorMaker 2 source and target integration enabled
+
+.. Warning::
+
+  If you experience connection problems to the source or target clusters, the MirrorMaker 2 service overview page shows the following error:
+  
+  .. image:: /images/tutorials/kafka-migration/error-connection.png
+    :alt: Aiven Console, MirrorMaker 2 service overview, error message stating that the connection to a target Kafka service is not working
+
+  You can also review the service logs by clicking on the **Log** tab.
+
+Start the MirrorMaker 2 replication flow
+''''''''''''''''''''''''''''''''''''''''
+
+In the previous steps you defined MirrorMaker 2 source and target aliases. Now it's time to define the replication flow.
+
+You first need to identify the set of topics you want to migrate to the new cluster, and create a :doc:`Java regular expression </docs/products/kafka/kafka-mirrormaker/concepts/replication-flow-topics-regex>` that includes them. As example, if you want to migrate all the topics starting with ``customer.`` your regular expression will be ``customer\..*``.
```

Comment:
Can we give an example, here, that includes more than one topic? (Regex is hard, examples are helpful, I think we can assume people will migrate more than one topic.)